src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
+
+  const beforeForEach = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item !== '');
+
+  beforeForEach.forEach((item) => {
+    const pair = item.split(':');
+    const key = pair[0].trim();
+    const value = pair[1].trim();
+
+    obj[key] = value;
```

Comment:
The 'forEach' loop is acceptable, but consider using more functional approaches like 'reduce' to directly create the object without the need for an intermediate variable.